Description: Fix man path for FHS
Author: Stefano Karapetsas <stefano@karapetsas.com>
--- a/setup.py
+++ b/setup.py
@@ -28,7 +28,7 @@
             'trash-empty   = trashcli.cmds:empty'
         ]
     },
-    data_files = [('man/man1', ['man/man1/trash-empty.1',
+    data_files = [('share/man/man1', ['man/man1/trash-empty.1',
                                 'man/man1/trash-list.1',
                                 'man/man1/trash-restore.1',
                                 'man/man1/trash-put.1'])],
